Bottom: 350cbc943ba1e9489b615cc35a5564b2f6a548ed
Top:    2163fcc372a3a6d54cfd6ff9386e7e8ee3b7a171
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-14 14:16:56 +0800

Refresh of net-core-introduce

---

diff --git a/include/net/sock.h b/include/net/sock.h
index 4f7c584..e05842b 100644
--- a/include/net/sock.h
+++ b/include/net/sock.h
@@ -783,6 +783,7 @@ enum sock_flags {
 	SOCK_FILTER_LOCKED, /* Filter cannot be changed anymore */
 	SOCK_SELECT_ERR_QUEUE, /* Wake select on error queue */
 	SOCK_RCU_FREE, /* wait rcu grace period in sk_destruct() */
+	SOCK_XDP_BUFF, /* accept a batch of XDP buffs */
 };
 
 #define SK_FLAGS_TIMESTAMP ((1UL << SOCK_TIMESTAMP) | (1UL << SOCK_TIMESTAMPING_RX_SOFTWARE))
